lib/traject/readers/marc_combining_reader.rb

```diff
@@ -0,0 +1,93 @@
+module Traject
```

Comment:
btw I mentioned that we are using this to Chris Beer at code4lib, told him thanks :)